README.asciidoc

```diff
@@ -172,6 +172,8 @@ Active
 * https://luakit.github.io/luakit/[luakit] (C/Lua, GTK+ with WebKit2)
 * http://surf.suckless.org/[surf] (C, GTK+ with WebKit1/WebKit2)
 * http://www.uzbl.org/[uzbl] (C, GTK+ with WebKit1/WebKit2)
+* https://github.com/next-browser/next/[next] (Lisp, GTK+ with Webkit2)
```

Comment:
I'm not sure if next uses webkit 2 or not. It looks like it is using webkit2 on linux, but on mac it seems to [be using legacy webkit](https://github.com/next-browser/next/issues/19#issuecomment-347832936) (at least at some point). I don't have a mac, so I can't test this, though.

Reply:
Yes, I found Linux data on their site, but not much about their Mac s/w (besides using Cocoa), and tried to match the format of other entries in that list.  I don't know much about Mac technology.
Let me know if I should make any changes to the text.

Reply:
FWIW I get this UA on macOS (El Capitan): Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/601.7.8 (KHTML, like Gecko)